data/json/furniture.json

```diff
@@ -598,7 +599,8 @@
             "items": [
                 { "item": "scrap", "count": [2, 6] },
                 { "item": "steel_chunk", "prob": 50 },
-                { "item": "pipe", "count": 1 }
+                { "item": "pipe", "count": 1 },
+                { "item": "lead", "charges": [500, 1000] }
```

Comment:
Yeah i was not sure what the philosophy is about that. I was just looking at it from a balance point of view to make taking the additional time for deconstruction attractive compared to just bashing it.